modules/op-release-notes-1-7.adoc

```diff
@@ -15,6 +15,18 @@ In addition to the fixes and stability improvements, the following sections high
 [id="pipelines-new-features-1-7_{context}"]
 === Pipelines
 
+* With this update, `pipelines-<version>` is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`. Cluster administrators can also use the `latest` channel to install the most recent stable version of the Operator. 
```

Comment:
Maybe use {pipelines-title} the first time and then switch to {pipelines-shortname}.